src/EditorFeatures/Core/Shared/Extensions/HostWorkspaceServicesExtensions.cs

```diff
@@ -61,8 +61,8 @@ private static string GetDefaultContentTypeName(HostWorkspaceServices workspaceS
 
             if (mefHostServices != null)
             {
-                var registry = mefHostServices.GetExports<IContentTypeRegistryService>().Single().Value;
-                return mefHostServices.GetExports<ILanguageService, ContentTypeLanguageMetadata>()
+                var exports = mefHostServices.GetExports<ILanguageService, ContentTypeLanguageMetadata>();
```

Comment:
'registry' was not used. Does the call to GetExports<IContentTypeRegistryService>() have side effects?